src/wawa-table.ts

```diff
@@ -155,6 +154,7 @@ export class WawaTable extends LitElement {
 
     public connectedCallback(): void {
         super.connectedCallback();
+        this.loadTemplates();
```

Comment:
moved here from constructor() to have access to members that are not available yet in the constructor